operator/gitops/argocd/pipeline-service/tekton-results/kustomization.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: tekton-results
 resources:
-  - https://github.com/openshift-pipelines/tektoncd-results.git/config/overlays/base-only/?ref=dd067cad783501dc87fdec7285ca62e573fd85e3
+  - https://github.com/avinal/tektoncd-results.git/config/overlays/base-only/?ref=7b41ef70aa658cf889ffdaf7ba0e73704f17ead6
```

Comment:
Why using your fork?

Reply:
I am testing it, this PR is not ready to merge, needs the results pr first.